e2e/tests/misc/sidenav.spec.ts

```diff
@@ -44,7 +44,7 @@ describe('Sidebar Navigation', () => {
 
     // Select Sign Out link
     await signOut(page);
```

Comment:
We should remove this entirely for now, this will affect other concurrent tests.